.github/workflows/gutenberg-packages-update.yml

```diff
@@ -159,19 +160,37 @@ jobs:
         if: steps.packages.outputs.list != 0
         run: |
           git add --all .
-          git commit -m "Update Gutenberg package dependencies"
-          git push origin "$HEAD_BRANCH"
+          git commit -m "Update WordPress NPM packages based on Gutenberg release v$VERSION" -m "Signed-off-by: $(git config user.name) <$(git config user.email)>"
+          git push -u origin "$HEAD_BRANCH"
         env:
+          VERSION: ${{ needs.check-gutenberg-release.outputs.latest-version }}
           HEAD_BRANCH: ${{ steps.branches.outputs.head }}
 
-      - name: Create pull request
+      - name: Draft a pull request
+        id: pr-create
         if: steps.packages.outputs.list != 0 && steps.remote-branch.outputs.exists == 0
         run: |
-          git push -u origin "$HEAD_BRANCH"
-          PR_URL=$(gh pr create --base "$BASE_BRANCH" --title "Update Gutenberg packages after v$VERSION release" --body "" --label dependencies | grep https://)
-          gh pr merge --auto --merge "$PR_URL"
+          PR_BODY=$(node bin/gutenberg-packages-update-pr-body.js "${{ steps.packages.outputs.list }}")
+          PR_URL=$(gh pr create --draft --base "$BASE_BRANCH" --title "Update Gutenberg packages after v$VERSION release" --body "$PR_BODY" --label dependencies | grep https://)
+          echo "::set-output name=pr-url::$PR_URL"
         env:
           VERSION: ${{ needs.check-gutenberg-release.outputs.latest-version }}
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
           BASE_BRANCH: ${{ steps.branches.outputs.base }}
           HEAD_BRANCH: ${{ steps.branches.outputs.head }}
+
+      # Make PR ready using personal access token to allow workflow to be triggered.
+      - name: Make PR ready
+        if: steps.pr-create.outputs.pr-url != ''
+        run: gh pr ready "$PR_URL"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_PAT }}
+          PR_URL: ${{ steps.pr-create.outputs.pr-url }}
+
+      - name: Add PR reviewers
+        if: steps.pr-create.outputs.pr-url != ''
+        # Add a PR reviewer so that a notification can be sent to the concerned person.
+        run: gh pr edit "$PR_URL" --add-reviewer ${{ secrets.PR_REVIEWER }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_URL: ${{ steps.pr-create.outputs.pr-url }}
```

Comment:
Is the PR reviewer necessary?

Reply:
The only idea behind this is to send a notification to a concerned person so that all changes on that PR can be tracked. LMK if the mentioned reason is feasible else I will remove this step.

Reply:
Got it. I am subscribed to all PRs regardless, so it doesn't really matter to me. Maybe just add a check for whether `secrets.PR_REVIEWER` is not empty. So then if the secret is added, the check will proceed. Otherwise, it can be skipped.

Reply:
I think I should remove it because inside conditionals we can not use `secrets` or `env` and hardcoding a reviewer name doesn't make much more sense. So I will remove this step.